src/teaching-element/Assessment/Feedback.vue

```diff
@@ -60,8 +60,10 @@ export default {
   methods: {
     order(index) {
       const assessmentType = camelCase(ASSESSMENT_TYPE[this.type]);
-      const order = this.options[assessmentType];
-      const orderType = order ? order.type : defaults[assessmentType].type;
+      const { type: orderType } = ({
+        ...defaults[assessmentType],
+        ...this.options[assessmentType]
+      });
```

Comment:
Make `config` computed variable that will return merged custom options with defaults (like we did on e.g. `MultipleChoice` and use it here (and throughout the file where possible).